cmake/kokkos_enable_devices.cmake

```diff
@@ -39,13 +49,16 @@ IF(KOKKOS_ENABLE_OPENMP)
   IF(KOKKOS_CLANG_IS_INTEL)
     SET(ClangOpenMPFlag -fiopenmp)
   ENDIF()
-  IF(KOKKOS_CXX_COMPILER_ID STREQUAL Clang AND "x${CMAKE_CXX_SIMULATE_ID}" STREQUAL "xMSVC")
-    #expression /openmp yields error, so add a specific Clang flag
-    COMPILER_SPECIFIC_OPTIONS(Clang /clang:-fopenmp)
-    #link omp library from LLVM lib dir
+  IF(KOKKOS_COMPILER_CLANG_MSVC)
+    #for clang-cl expression /openmp yields an error, so directly add the specific Clang flag
+    SET(ClangOpenMPFlag /clang:-fopenmp=libomp)
```

Comment:
This got added in the Travis build now, and thus fails.

Reply:
Ok I see the error, will change it. Thanks.